packages/manager/apps/web/client/app/hosting/database/DATABASE.html

```diff
@@ -67,7 +67,7 @@
                     <li>
                         <a
                             data-translate="hosting_tab_DATABASES_configuration_order_title_button_private"
-                            data-ui-sref="app.hosting.dashboard.database.order-private"
+                            href="#/order-cloud-db"
```

Comment:
it's a better practice in angular js to use the ui-sref directive to generate the link with your associated ui router state, in your case is it something like ui-sref="app.private-database-order-clouddb" ?

Reply:
Hello, thanks for the review. I updated the link